bindings/netstandard/ElectionGuard/ElectionGuard.ElectionSetup.Tests/KeyCeremony/TestKeyCeremony.cs

```diff
@@ -24,8 +24,11 @@ public void Test_KeyCeremony()
             var guardianId = sequenceOrder.ToString();
             var random = new Random((int)sequenceOrder);
             var guardian = new Guardian(
-                guardianId, sequenceOrder, numberOfGuardians,
-                quorum, keyCeremony.Id,
+                guardianId, sequenceOrder,
+                numberOfGuardians,
+                quorum,
+                CiphertextElectionContext.ParameterBaseHash,
```

Comment:
I considered using a default value throughout in the interface but decided against that and to force the application to provide this value because it is runtime dependent (even though it is a compiled constant). This way, the caller is forced to understand that we have a dependency on the values G,Q, and P that are used to make the guardian. Really this probably should be part of the key ceremony and then the guardian can derive agains the key ceremony but i was trying to minimize the number of changes.